godot-core/src/obj/gd.rs

```diff
@@ -90,6 +90,15 @@ use crate::{classes, out};
 ///
 /// For type conversions, please read the [`godot::meta` module docs][crate::meta].
 ///
+/// # Exporting
+///
+/// The [`Export`][crate::registry::property::Export] trait is not directly implemented for objects `Gd<T>`, because Godot objects can be
+/// nullable, while `Gd<T>` cannot. Instead, `Export` is implemented for [`OnEditor<Gd<T>>`][crate::obj::OnEditor], which validates that objects
```

Comment:
maybe something along lines of:

```suggestion
/// The [`Export`][crate::registry::property::Export] trait is not directly implemented for `Gd<T>`, because editor requires object-based properties to be
/// nullable, while `Gd<T>` can't be null. Instead, `Export` is implemented for [`OnEditor<Gd<T>>`][crate::obj::OnEditor], which validates that objects
```
?